### ISSUE TITLE
Can't display parameter name with body type (Spec 1.2)

### ISSUE THREAD

**Issue by birbird** [swagger-api/swagger-ui#1493]
Here is the swagger json
```
      "operations": [
        {
          "notes": "",
          "summary": "",
          "method": "POST",
          "parameters": [
            {
              "paramType": "form",
              "name": "access_token",
              "description": null,
              "type": "string",
              "required": true,
              "allowMultiple": false
            },
            {
              "paramType": "form",
              "name": "circle[name]",
              "description": "",
              "type": "string",
              "required": true,
              "allowMultiple": false
            },
            {
              "paramType": "body",
              "name": "circle[picture]",
              "description": "",
              "type": "File",
              "required": false,
              "allowMultiple": false
            },
            {
              "paramType": "body",
              "name": "circle[member_profile_itmes]",
              "description": null,
              "type": "String",
              "required": false,
              "allowMultiple": false
            }
          ],
          "type": "void",
          "consumes": [
            "multipart/form-data"
          ]
        }
    ]
```

The swagger UI shows 
![circles](https://cloud.githubusercontent.com/assets/770541/8890164/c2e4494a-3328-11e5-9aa9-ef8efbab2320.png)

The parameters with paramType show their name is body.
And older version (tag v2.0.24) can show the names in json.

Is this a bug or expected?

**Comment by webron** [comment]
@birbird - The current version of the UI converts older Swagger definitions to 2.0. As part of the conversions, some info can get lost/mistranslated, especially if the origin was an malformed definition.

Unfortunately, the sample you provided is malformed for three reasons:
- In 1.2, technically, `body` parameters have to have the name `body`. Other names are not supported.
- You can have only one `body` parameter. This is true for 2.0 as well.
- You can't mix `body` parameters with `form` parameters. Just as above, this is true for 2.0 as well.

In short, there's very little the UI can do with your definition, unfortunately.

**Comment by birbird** [comment]
@webron Thanks a lot for kindly explaination!
